### PR TITLE
Update quality manual policy objectives

### LINKED ISSUE #63: General software quality measures in `0-quality-manual-policy-objectives.md`
This only addresses the quality of the "algorithm".
It does not address the quality of the software (defects, uptime, customer satisfaction related to non-functional requirements, such as performance, ease of use, ease of installation, etc.)

_Originally posted by @sjswerdloff in https://github.com/RadiotherapyAI/RadiotherapyAI/pull/61#discussion_r973979973_

### LINKED ISSUE #64: Put roles on separate lines in `docs/1-qms/0-quality-manual-policy-objectives.md`
Is there a reason why you aren't putting each role on a separate line and listing your name for each one.
At some point, and certainly as a copyable example, one would expect each role to be filled by a different person, or at least not all of them filled by one person.

_Originally posted by @sjswerdloff in https://github.com/RadiotherapyAI/RadiotherapyAI/pull/61#discussion_r973981338_

### PR DESCRIPTION
**Pull request by SimonBiggs**
Fixes #64 and fixes #63

### CODE REVIEW

File: docs/1-qms/0-quality-manual-policy-objectives.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ being stand-alone software:
 > which are important for it. Maybe you're developing software for patients
 > with a certain disease and your goal is to improve their lives. -->
 
-{{device_manufacturer}} aims to make AI assisted radiotherapy cancer treatments
-accessible to all. With the goals to reduce treatment errors while improving
-overall treatment efficacy and efficiency.
+{{device_manufacturer}} aims to use AI to save lives within radiotherapy. To
+achieve this it has the goals to reduce treatment errors and improve treatment
+efficacy while helping more patients be treated through efficiency gains.
```
Comment (sjswerdloff) [inline docs/1-qms/0-quality-manual-policy-objectives.md:70]: more patients **to** be treated

File: docs/1-qms/0-quality-manual-policy-objectives.md [hunk 2]
```diff
@@ -97,17 +117,24 @@ comparisons are undergone utilising the following metrics:
...
-| CEO, QMO, PRRC, CTO, CPO, Software Engineer | Simon Biggs      |
-| Regulatory Consultant                       | Dr Oliver Eidel  |
-| Regulatory Consultant                       | Pierre Lonchampt |
+| Role(s)               | People           |
+| --------------------- | ---------------- |
+| CEO                   | Simon Biggs      |
+| QMO                   | Simon Biggs      |
+| PRRC                  | Simon Biggs      |
+| CTO                   | Simon Biggs      |
+| Advisor               | Stuart Swerdloff |
```
Comment (sjswerdloff) [inline docs/1-qms/0-quality-manual-policy-objectives.md:135]: very thoughtful of you!

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
It might be a difference in Commonwealth English vs. USA 
but I think the grammar warrants the change request.

File: docs/1-qms/0-quality-manual-policy-objectives.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/1-qms/0-quality-manual-policy-objectives.md:70]:
```suggestion
efficacy while helping more patients to be treated through efficiency gains.
```

File: docs/1-qms/0-quality-manual-policy-objectives.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/1-qms/0-quality-manual-policy-objectives.md:135]: I wanted to check, is this okay with you?

Comment (sjswerdloff) [reply docs/1-qms/0-quality-manual-policy-objectives.md:135]: Definitely.

**Review by sjswerdloff — APPROVED** [review]

File: docs/1-qms/0-quality-manual-policy-objectives.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/1-qms/0-quality-manual-policy-objectives.md:135]: 😄

**Pull request merged**